cortex-m-openocd.Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ghcr.io/modm-ext/modm-build-base:TAG
+LABEL maintainer="Niklas Hauser <niklas.hauser@rwth-aachen.de>, Raphael Lehmann <raphael+docker@rleh.de>, Christopher Durand <christopher.durand@rwth-aachen.de>"
```

Comment:
Feel free to add yourself as a maintainer!

Reply:
do I really have to? :D

Reply:
We will forward problems and support for the OpenOCD stuff to you anyway. 😉